guide/README.md

```diff
@@ -455,6 +455,69 @@ used to bump the `Mi 4` note up by one octave and then to bump the
 | | | For example: Transposing C-D-E-F by 4 (fifth) will give us G-A-B-C
 | | | To transpose an octave: shift by the mode length (7 in major scales) up or down. |
 
+#### <a name="Set Key"></a>3.2.5 Set Key
+
+| Set Key Example |
+| --- |
+| ![set key](./setkey1.svg "set key example") |
+
+
+The *Set key* block is used to set the key and mode, e.g, C Major. 
+It is used to change both the mode and key of the current scale. (The current scale is used to define the mapping of Solfege to notes and also the number of half steps take by the the Scalar step block.
+
+The example above demonstrates traveling up and down the major
+ scale by moving an octave up from the starting note,
+ do, one note at a time and then back down the same way.
+
+
+| Standard Notation for Set Key Example |
+| --- |
+| ![set key notation](./setkey2.png "Standard Notation Set Key") |
+
+
+
+[RUN LIVE](https://musicblocks.sugarlabs.org/index.html?id=1662103714150464&run=True)
+
+When using the *Set key* block, the mode argument is used to define the pattern of half steps and the key argument is used to define the starting position of the pattern. For example, when mode = "major" and key = "C", the pattern of half steps is 2 2 1 2 2 2 1 and the first note in the scale from which the pattern is applied is "C".
+
+Examples are shown in the following table.
+
+| Key | Mode | Notes Pattern | Modes Pattern |
```

Comment:
Please switch the order of the 3rd and 4th columns. (The notes pattern is the output, so it should be at the end.)

Reply:
done